<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HtmlPageSearcherClass</class>
 <widget class="QMainWindow" name="HtmlPageSearcherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HtmlPageSearcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>210</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="m_search_text_label">
         <property name="text">
          <string>Search text:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="m_search_text_edit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="m_start_url_label">
         <property name="text">
          <string>Start URL:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="m_start_url_text_edit">
         <property name="plainText">
          <string>http://stackoverflow.com</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="m_max_urls_number_label">
         <property name="text">
          <string>URL max number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_url_max_number_spin_box">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="m_threads_number_label">
         <property name="text">
          <string>Threads number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_threads_number_spin_box">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>451</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="m_button_stop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_button_pause">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_button_start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEdit" name="m_text_edit"/>
       </item>
       <item>
        <widget class="QProgressBar" name="m_progress_bar">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HtmlPageSearcher.qrc"/>
 </resources>
 <connections/>
</ui>
